scripts/components/Interactions/NavigationButton.js

```diff
@@ -5,6 +5,7 @@ import './NavigationButton.scss';
 import {H5PContext} from "../../context/H5PContext";
 import NavigationButtonLabel, {getLabelPos, getLabelText, isHoverLabel} from "./NavigationButtonLabel";
 import HotspotNavButton from "./HotspotNavButton";
+import {staticWidth, staticHeight} from "../Scene/SceneTypes/StaticScene";
```

Comment:
Brukes disse i denne fila? Om ikkje trengs ikkje importen.
```suggestion
import { staticWidth, staticHeight } from "../Scene/SceneTypes/StaticScene";
```